Docs/articles/Kernel/CGS.md

```diff
@@ -72,38 +74,32 @@ namespace GraphicTest
                 pen.Color = Color.PaleVioletRed;
                 canvas.DrawRectangle(pen, 350, 350, 80, 60);
 
-                Console.ReadKey();
-
-                /* Let's try to change mode...*/
-                canvas.Mode = new Mode(800, 600, ColorDepth.ColorDepth32);
-
                 /* A LimeGreen rectangle */
                 pen.Color = Color.LimeGreen;
                 canvas.DrawRectangle(pen, 450, 450, 80, 60);
 
-                Console.ReadKey();
```

Comment:
Doesnt this cause the kernel to instantly close? I think its better if we keep it.

Reply:
But it's a Console function..

Reply:
Well its used to get a key input, at this point I think its ok to use, since it should just block the os until the user wants to close kernel.